### DISCUSSION TITLE
Project Planning

### DISCUSSION THREAD

**Discussion by BhabishyaLuitel** [BhabishyaLuitel/Want2Know#3] [Announcements]
Let's start with the design of the project.

**Comment by AayamKhatiwada** [comment]
Path to the UI design: https://www.figma.com/file/7ZUfOgd0REXew0M7vEmLqf/Free-Blog-Website-Template-(Community)?type=design&node-id=0-1&mode=design&t=8niqAEVZws66yShQ-0
